onnx/defs/schema.cc

```diff
@@ -18,9 +18,15 @@
 #include "onnx/common/stl_backports.h"
 
 namespace ONNX_NAMESPACE {
+// -1 means ONNX schema hasn't been loaded yet
+// 0 means all versions of ONNX schema have been loaded
+// Other positive integer means the latest ONNX schemas before that version have been loaded
```

Comment:
change comment to : Other positive integer means the ONNX schemas for the specified version have been loaded

Reply:
Yes, I agree, I was confused about the intention.